src/App.tsx

```diff
@@ -20,17 +27,18 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={(event) => {
+                  setQuery(event.target.value);
+                }}
```

Comment:
create a separate function for this
```suggestion
                onChange={handleChange}
```